mwclient/client.py

```diff
@@ -421,7 +499,19 @@ def raw_call(self, script, data, files=None, retry_on_error=True, http_method='P
                 sleeper.sleep()
 
     def raw_api(self, action, http_method='POST', *args, **kwargs):
-        """Send a call to the API."""
+        """Send a call to the API.
+
+        Args:
+            action (str): The MediaWiki API action to perform.
+            http_method (str): The HTTP method to use in the request.
+
+        Returns:
+            The API response.
+
+        Raises:
+            errors.APIDisabledError: The MediaWiki API is disabled for this instance.
+            errors.InvalidResponse: The API response could not be decoded from JSON.
+        """
```

Comment:
it can raise various other things, via `raw_call`: at least `requests.exceptions.HTTPError` and `mwclient.errors.MaximumRetriesExceeded`.

Reply:
I've pushed an updated docstring, adding `errors.MaximumRetriesExceeded`, `requests.exceptions.HTTPError`, `requests.exceptions.ConnectionError`, and `requests.exceptions.Timeout`.